test/visualTest.ts

```diff
@@ -231,6 +231,22 @@ module powerbi.extensibility.visual.test {
                             expect(element.css("font-size")).toBe(expectedFontSize);
                         });
                 });
+
+                it("animation", (done) => {
```

Comment:
Name of the test isn't clear to me. What does this test check? I suppose that it check an ability to turn animation off. Is that correct?